File: app/database/models/Robotica.js
module.exports = (sequelize, dataTypes) => {
    let alias = 'Robotica';
    let cols = {
        id: {
            type: dataTypes.INTEGER.UNSIGNED,
            primaryKey: true,
            autoIncrement: true,
            allowNull: false
        },
        name: {
            type: dataTypes.STRING(50),
            allowNull: false
        },
        country: {
            type: dataTypes.STRING(50)
        },
        activity_start: {
            type: dataTypes.INTEGER
        },
        activity_end: {
            type: dataTypes.INTEGER
        },
        fuel: {
            type: dataTypes.STRING(50)
        },
        research: {
            type: dataTypes.STRING(50)
        }
    };
    let config = {
        tableName: 'naves_roboticas',
        timestamps: false
    }
    const Robotica = sequelize.define(alias, cols, config); 
 
    return Robotica
};

File: spaceStation/src/main/java/challenge/spacestation/data/modelos/Robotica.java
package challenge.spacestation.data.modelos;

/**
 *
 * @author mcami
 */
public class Robotica extends Nave{
    public String research;

    public Robotica() {
    }

    public Robotica(String research, int id, String name, String country, int activity_start, int activity_end, String fuel) {
        super(id, name, country, activity_start, activity_end, fuel);
        this.research = research;
    }

    public String getResearch() {
        return research;
    }

    public void setResearch(String research) {
        this.research = research;
    }
}
